### ISSUE TITLE
RememberStatePerDocument=true only acts if RememberOpenedFiles=true

### ISSUE THREAD

**Issue by guigonse** [sumatrapdfreader/sumatrapdf#667]
Could it be a regression as in [issue #7 ](https://github.com/sumatrapdfreader/sumatrapdf/issues/7) ?

**Comment by kjk** [comment]
This is by design. RememberOpenedFiles dictates whether we save this information at all.

RememberStatePerDocument dictates how much information per file we store.

**Comment by guigonse** [comment]
Thank you for the quick answer.
Just comment that in previous versions I could had `RememberStatePerDocument=true` with `RememberOpenedFiles=false`, so I can open each file in the same state I left it, but without the behaviour of opening all the files I had in my previous session of SUMATRA (which probably corresponded to a different project), which is what I obtain activating `RememberOpenedFiles`.
Congratulations for the program.

**Comment by kjk** [comment]
Hmm, it might be a regression then. Thanks for additional info.

**Comment by guigonse** [comment]
You're welcome! Congrats again.

**Comment by guigonse** [comment]
Sorry, but I've discovered my explanation wasn't right: the behavior that was annoying me in version 3.1.2 x64 was that, having  `RememberStatePerDocument=true`  and  `RememberOpenedFiles=true` , when opening  a single PDF Sumatra opens all PDFs opened in previous session plus the one selected, which wasn't the behavior until version 3.0, despite having exactly the same variable setting.
So, until version 3.0 -included- Sumatra acted like having `RememberOpenedFiles` allways switched off, and in 3.1 version it acts like having `RememberOpenedFiles` allways switched on.
I've reinstalled Sumatra version 3.0 and double checked it in my Windows 7 Pro x64 just now.
Best regards.
